proxy/proxy.go

```diff
@@ -751,34 +751,35 @@ func (p *Proxy) makeBackendRequest(ctx *context) (*http.Response, *proxyError) {
 }
 
 // checkRatelimit is used in case of a route ratelimit
-// configuration. It returns the used ratelimit.Settings and true if
+// configuration. It returns the used ratelimit.Settings, true and 0 if
```

Comment:
what does the 3 parameter do?
Right now for me this sounds like a duplication of the 2nd return argument.
Can't we drop the bool and use int instead to make this working?